src/typings/jwt.typing.ts

```diff
@@ -0,0 +1,7 @@
+export type JWT = {
```

Comment:
For a singular case is usually to have `file.service.ts` because normally you have a single class, in this case you have a lot of types, what if you use `jwt.typings` instead of `jwt.typing`?

Reply:
Resolved on 405d4653a891f3960b46d94eb9d29f2ce3d075de.